components/jenkins/local.tf

```diff
@@ -0,0 +1,5 @@
+locals {
```

Comment:
group locals close to they are used to make them easier to find and reason about.
you only need to move them to a separate file if they are used in multiple places